skeleton/webpack/webpack.config.js

```diff
@@ -21,7 +21,23 @@ const testDir = path.resolve(__dirname, 'test', 'unit');
 const nodeModulesDir = path.resolve(__dirname, 'node_modules');
 const baseUrl = '/';
 
+// @if feat['postcss-tailwind']
+const purgecss = require('@fullhuman/postcss-purgecss')({
+  content: [
+    path.resolve(srcDir, '/**/*.html')
+  ],
+  defaultExtractor: content => content.match(/[A-Za-z0–9-_:/]+/g) || []
+});
+// @endif
+
+// @if ! feat['postcss-tailwind']
 const cssRules = [
+// @endif
+// @if feat['postcss-tailwind']
+const cssRules = (production) => [
```

Comment:
Why not make `cssRules` a function no matter whether it's for tailwind or not? So that you don't need additional condition on all css/less/scss loader config.

BTW, turn off cssnano for non-production build is a reasonable for config without tailwind too.

Reply:
fixed